src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -539,14 +539,27 @@ public byte[] getAttestationChallenge(long devicePtr) {
     return getAttestationChallenge(deviceControllerPtr, devicePtr);
   }
 
-  /** Subscribe to the given attribute path. */
+  /**
+   * @brief Auto-Resubscribe to the given attribute path with keepSubscriptions and isFabricFiltered
+   * @param SubscriptionEstablishedCallback Callback when a subscribe response has been received and
+   *     processed
+   * @param ResubscriptionAttemptCallback Callback when a resubscirption haoppens, the termination
+   *     cause is provided to help inform subsequent re-subscription logic.
+   * @param ReportCallback Callback when a report data has been received and processed for the given
+   *     paths.
+   * @param devicePtr connected device pointer
+   * @param attributeList a list of attribute paths
```

Comment:
1. Require to add minInterval, maxInterval.
2. Require to delete ResubscriptionAttemptCallback
3. attributeList -> attributePath

Reply:
done, thanks